README.md

```diff
@@ -40,6 +40,8 @@ resources:
 | ---- | ------ | ------------ | -------------------------- |
 | type | string | **Required** | `custom:text-divider-row`  |
 | text | string | **Required** | Text to display in divider |
+| margin | string or number | Optional | Custom margin CSS |
```

Comment:
These should be CSS variables, not card options

Reply:
Hi @iantrich,

I agree this would make more sense, but would it not then restrict this to one configuration/style per theme i.e. a site-wide change, rather than allowing for customisation for each instance of the widget.

I guess you could use something like card-mod with it, but I thought this might be a bit overkill.

Happy to make the changes if you disagree.

Best,
Ryan

Reply:
Theme can be defined at the card-level. In it's current format, yes, it is somewhat overkill as we don't really have a theme editor or easier way to tweak things on the fly, but I think long-term it makes more sense. Ideally we'll have a theme editor in the UI before long and custom cards can expose their custom CSS attributes that change the look/feel there.

Reply:
Hey @iantrich,

No worries, have implemented the requested changes.

Best,
Ryan